sphinx/tutorial/rose/furthertopics/rose-stem-tutorial.rst

```diff
@@ -267,15 +270,24 @@ Now edit the file::
 
    rose-stem/app/spaceship/rose-app.conf
 
-and change one of the thrusts, then rerun ``rose stem``. You will find the
-``rose_ana_position`` task fails, as the results have changed.
+and change one of the thrusts, then rerun ``rose stem`` and
+``cylc play spaceship_working_copy/run2``.
+
+
+You will find the ``rose_ana_position`` task fails, as the results have changed.
 
-.. TODO - Do we need to reset previous changes?!
```

Comment:
I checked this. It is not necessary to delete the changes. You can tell there are more failed tests (3 rather than 1).

Reply:
```suggestion
```